tests/ert_tests/ert3/config/test_experiment_config.py

```diff
@@ -10,11 +10,16 @@ def test_valid_evaluation():
     assert experiment_config.algorithm == None
 
 
-def test_valid_sensitivity():
-    raw_config = {"type": "sensitivity", "algorithm": "one-at-a-time"}
+@pytest.mark.parametrize(
```

Comment:
We should also have a test ensuring that passing no tail also is a valid configuration.